src/components/Configure/nav/ObjectManagementNav/constant.tsx

```diff
@@ -1,2 +1,4 @@
-export const OTHER_CONST = 'other';
+// WRITE_CONST is used for the "Write" tab, its value is "other" for legacy reasons.
+export const WRITE_CONST = 'other';
```

Comment:
you can probably rename 'other' to 'write' here. I don't see 'other' hard coded anywhere here. just references to the enum.